backend/prompt_studio/prompt_studio_registry/exceptions.py

```diff
@@ -14,3 +14,15 @@ class ToolDoesNotExist(APIException):
 class ToolSaveError(APIException):
     status_code = 500
     default_detail = "Error while saving the tool."
+
+
+class EmptyToolExportError(APIException):
+    status_code = 500
+    default_detail = "Empty tool without prompts cannot be exported. \
+        Try adding a prompt and executing it."
```

Comment:
@harini-venkataraman did you try reproducing this error with the default detail? I think this might add a new line in the string - I've suggested a safer convention to split long strings
```suggestion
    default_detail = (
        "Empty Prompt Studio project without prompts cannot be exported. "
        "Try adding a prompt and executing it."
    )
```

Reply:
@chandrasekharan-zipstack Yes, a new line was added, it gave a good readability. But yes, let us go with safer conventions.